samples/projection-management.ts

```diff
@@ -264,7 +264,7 @@ describe("[sample] projection-management", () => {
     const name = await createTestProjection();
 
     // region GetStatus
-    const projection = await client.getProjectionStatistics(name);
+    const projection = await client.getProjectionStatus(name);
```

Comment:
**CONCERN:** This will be a breaking change, and we just released the major version. Could we provide an overload and mark the old one as obsolete?